Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -503,9 +603,47 @@ extension _FileManagerImpl {
             #endif
             return attributes
         }
+#endif
     }
     
     func attributesOfFileSystem(forPath path: String) throws -> [FileAttributeKey : Any] {
+#if os(Windows)
+        return try path.withNTPathRepresentation { pwszPath in
+            let dwLength: DWORD = GetFullPathNameW(pwszPath, 0, nil, nil)
```

Comment:
`withNTPathRepresentation` already calls `GetFullPathNameW` right? If so, is there an equivalent of `strlen` we could use here instead that might be faster?

Reply:
Not in the case that you have a NT path:

```swift
return try path.withCString(encodedAs: UTF16.self) { pwszPath in
  guard !path.hasPrefix(#"\\"#) else { return try body(pwszPath) }
  ...
}
```